src/Billing/Enum/PreferenceTypeEnum.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Billing\Enum;
+
+use ArrowSphere\PublicApiClient\AbstractEnum;
+
+abstract class PreferenceTypeEnum extends AbstractEnum
```

Comment:
If this class is abstract, please name it AbstractPreferenceTypeEnum
See https://www.php-fig.org/bylaws/psr-naming-conventions/